webui/react/src/components/TaskActionDropdown.tsx

```diff
@@ -53,6 +53,9 @@ const TaskActionDropdown: React.FC<Props> = ({ task }: Props) => {
       switch (params.key) { // Cases should match menu items.
```

Comment:
nit: Not part of this PR, but up do you. Move `// Cases should match menu items.` to the line above?

Reply:
I prefer it here but the line above works too.
I move this section in #652 or and they'll collide if I change so slightly rather not change it here